bridges/CssSelector2Bridge.php

```diff
@@ -0,0 +1,421 @@
+            ],
+            'url_pattern' => [
+                'name' => '[Optional] Pattern for site URLs to keep in feed',
+                'title' => 'Optionally filter items by applying a regular expression on their URL',
+                'exampleValue' => '/blog/article/.*',
+            ],
+            'limit' => self::LIMIT,
+            'content_cleanup' => [
+                'name' => '[Optional] Content cleanup: List of items to remove',
+                'title' => 'Selector for unnecessary elements to remove inside article contents.',
+                'exampleValue' => 'div.ads, div.comments',
+            ],
+            'title_selector' => [
+                'name' => '[Optional] Selector for the article title',
+                'title' => 'Selector to select the article title',
+                'defaultValue' => 'h1'
+            ],
+            'title_cleanup' => [
+                'name' => '[Optional] Text to remove from feed title',
+                'title' => <<<EOT
+                Text to remove from the feed title, which is read from the article list page.
+                EOT,
+                'exampleValue' => ' | BlogName',
+            ],
+            'category_selector' => [
+                'name' => '[Optional] Categories',
+                'title' => <<<EOT
+                Selector to extract the catgories the article has
+                EOT,
+                'exampleValue' => 'span.category, #main-category'
+            ],
+            'author_selector' => [
+                'name' => '[Optional] Author',
+                'title' => <<<EOT
+                Selector to extract the author of the article. If multiple elements are selected
+                the first one is used.
+                EOT,
+                'exampleValue' => 'span#author'
+            ],
+            'time_selector' => [
+                'name' => '[Optional] Time selector',
+                'title' => <<<EOT
+                Element that contains a php `strtotime` compatible text. If the element 
+                is a `time` element, the value for the `datetime` attribute is used.
+                EOT,
+            ],
+            'use_article_pages' => [
+                'name' => 'Load article from page',
+                'title' => <<<EOT
+                If true, the article page is load and parsed to get the article contents using 
+                the css selectors. (Slower!)
+                Otherwise, the element selected by the article entry selector is used.
+                EOT,
+                'type' => 'checkbox'
+            ],
+            'article_page_content_selector' => [
+                'name' => '[Optional] Selector to select article page content',
+                'title' => 'Extract the article from its page using the provided selector',
+                'exampleValue' => 'article.content',
+            ],
+            'remove_styling' => [
+                'name' => 'Remove styling',
```

Comment:
```suggestion
                'name' => '[Optional] Remove styling',
```
(I know that checkboxes are always optional but otherwise this one will stand out)